### PR TITLE
feat: [IDP-58] Partition tuning

### PR DESCRIPTION
**Pull request by marco-di-lisa**
### List of changes

Increased partition number for some idpay topics

### Motivation and context

Performance

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

#### Has This Been Tested?

- [ ] Yes
- [x] No

### Other information
Target: 
```

```

#### How Has This Been Tested?

- [ ] Unit Test Suite
- [ ] Integration Test Suite
- [ ] Api Tests
- [ ] Load Tests

[Link to test results](https://pagopa.atlassian.net/browse/RTD-XXXX)

---

### CODE REVIEW

File: src/core/env/uat/terraform.tfvars [hunk 1]
```diff
@@ -592,7 +592,7 @@ eventhubs = [
   },
   {
     name              = "rtd-trx"
-    partitions        = 1
+    partitions        = 16
```
Comment (petretiandrea) [inline src/core/env/uat/terraform.tfvars:595]: Could you create a different PR for RTD changes?

Comment (GiovanniMancini) [inline src/core/env/uat/terraform.tfvars:595]: This change should  be moved in another PR.. the queue is hosted in another domain

**Review by GiovanniMancini — CHANGES_REQUESTED** [review]
Change to rtd-trx should be moved in another PR/branch.

File: src/core/env/uat/terraform.tfvars (hunk 1 above)
Comment (marco-di-lisa) [reply src/core/env/uat/terraform.tfvars]: Moved change to https://github.com/pagopa/cstar-infrastructure/pull/857
